lib/src/helpers/target_platform_extension.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:io';
+
+import 'package:flutter/foundation.dart';
+import 'package:meta/meta.dart';
+
+@internal
+extension IsRuntimePlatform on TargetPlatform {
+  bool get isRuntimePlatform {
```

Comment:
Tests are intended to be runned on linux macOS or window.
Maybe you could log a warning if `TargetPlatform` is something else.

Reply:
I've added a log